Src/LiquidProjections.PollingEventStore/PollingEventStoreAdapter.cs

```diff
@@ -136,274 +194,253 @@ private Page TryGetNextPageFromCache(long precedingCheckpoint, string subscripti
                     }
                     else
                     {
-                        StartPreloadingNextPage(lastCheckpoint, subscriptionId);
+                        Task _ = RequestPageload(subscriptionId, lastCheckpoint, cancellationToken);

```

Comment:
❓ There is already code to prefetch a page, why add additional code for it here?